test/ssh_authorized_keys_test.rb

```diff
@@ -29,69 +31,78 @@
   let(:invalid_key_spec) { double("invalid_key") }
   let(:home) { "/home/user" }
   let(:key) { double("key") }
+  let(:keys) { [key] }
 
-  before do
-    allow(Yast::Users::SSHAuthorizedKey).to receive(:build_from)
-      .with(valid_key_spec)
-      .and_return(key)
-    allow(Yast::Users::SSHAuthorizedKey).to receive(:build_from)
-      .with(invalid_key_spec)
-      .and_return(nil)
-    ssh_authorized_keys.main
-  end
+  before { subject.main }
```

Comment:
OK, then please add a comment why, this is quite unusual in YaST. Or add a `reset` method to be clear...